azurelinuxagent/common/event.py

```diff
@@ -393,52 +529,106 @@ def _clean_up_message(message):
             else:
                 return message
 
+    def _add_common_event_parameters(self, event, event_timestamp):
+        """
+        This method is called for all events and ensures all telemetry fields are added before the event is sent out.
+        Note that the event timestamp is saved in the OpcodeName field.
+        """
+        common_params = [TelemetryEventParam('GAVersion', CURRENT_AGENT),
+                         TelemetryEventParam('ContainerId', GoalState.ContainerID),
+                         TelemetryEventParam('OpcodeName', event_timestamp.strftime(u'%Y-%m-%dT%H:%M:%S.%fZ')),
```

Comment:
#1757 changed the format of the timestamp. I talked to @pgombar about this and she confirmed it was not intentional. I was going to revert to the previous format but 2.2.46 was just converting the datetime to string, with no formatting. This new format is safer, and plays well in Kusto queries, so I left it alone. Just be aware of the change from 2.2.46 in case it ever matters.